pages/[network]/avatar/[address].tsx

```diff
@@ -1,10 +1,13 @@
 import type { NextPage, GetServerSideProps } from "next";
-import { getBalance, getAvatar } from "../../../apiClient";
+import { getBalance } from "../../../apiClient";
 import { getNetworkType, getNodeType } from "../../../utils/network";
 import { getSdk } from "../../../utils/mimirGraphQLClient";
+import { NetworkType, NodeType } from "../../../constants/network";
 
-const CURRENCY_TICKERS = [
+const AGENT_CURRENCY_TICKERS = [
     "CRYSTAL",
+];
```

Comment:
If you didn't want to the array dynamically, it will be better to make the constant array stay in readonly state.

```suggestion
const AGENT_CURRENCY_TICKERS = [
    "CRYSTAL",
] as const;
```

Reply:
Thanks for the advice!